src/components/AssetAccountDetails.tsx

```diff
@@ -30,18 +29,6 @@ export const AssetAccountDetails = ({ assetId, accountId }: AssetDetailsProps) =
   const marketData = useAppSelector(state => selectMarketDataById(state, assetId))
   const assetIds = useMemo(() => [assetId], [assetId])
 
-  const clearAmounts = useSwapperStore(state => state.clearAmounts)
-  const updateBuyAsset = useSwapperStore(state => state.updateBuyAsset)
-
-  const feeAsset = useAppSelector(state => selectFeeAssetById(state, assetId))
-  const buyAsset = useAppSelector(state => selectAssetById(state, assetId))
-
-  useEffect(() => {
-    const buyAssetWithFallback = buyAsset ?? feeAsset
-    if (buyAssetWithFallback) updateBuyAsset(buyAssetWithFallback)
-    clearAmounts()
```

Comment:
By removing this I was able to observe inconsistent behaviour, whereby now we sometimes clear the amounts when changing routes, and sometimes we don't.

We might need to consider the product implications of not clearing amounts between pages, as we don't want stale quotes floating around.

We did have a quote retry mechanism in place, though I'm not sure it is still working.

Reply:
yeah we are actually doing this in prod where quotes values are left as-in in the trade input state until something is changed. Only hitting a page where a default is provided will clear the state, which is the logic here too. But I agree that to the user this appears inconsistent and we dont have solid logic to refresh quotes. 

I'm going to make the trade input clear the values on every load for consistency.

Reply:
Yeah right. If it's in prod we should for sure get that fixed - double whammy PR 🙏